20FinalProject/index.html

```diff
@@ -1,10 +1,181 @@
+              <li>
+                <a href="#services">Services</a>
+              </li>
+              <li>
+                <a href="#portfolio">Portfolio</a>
+              </li>
+              <li>
+                <a href="#contact">Contact</a>
+              </li>
+            </ul>
+          </div>
+          <!-- /.navbar-collapse -->
+        </div>
+        <!-- /.container -->
+      </nav>
+      <header>
+        <h1>Kira Villarreal</h1>
+        <h4>Web Design and Other Cool Stuff</h4>
+      </header>
+      <section class="about" id="about">
+        <h2>About Me</h2>
+        <img src="https://kiravillarreal.github.io/intro-workbook/20FinalProject/img/kira.jpg" alt="Kira" />
+        <p>I'm a professional website designer with about 70 days worth of experience, thanks to my awesome education at Austin Coding Academy. Before that, I pretty much didn't know how to turn on a computer. I also have more than 15 years of experience as a communications professional, copywriter, and commerical voiceover artist. I can design and build your website from start to finish, and if you need help creating the content and advertising your services, I can do that for you too! Thanks so much for visiting my site. I'm looking forward to helping you grow your business! </p>
+      </section>
+      <section class="services" id="services">
+        <h2>Services</h2>
+        <div class="servicebox">
+          <article>
+            <img class="animated infinite pulse" src="https://kiravillarreal.github.io/intro-workbook/20FinalProject/img/web-dev.png" alt="webdesign" />
+            <h4>Website Creation</h4>
+            <p>I create responsive websites to showcase your business. Check out some examples in my portfolio section below.</p>
+          </article>
+          <article>
+          <img class="animated infinite pulse" src="https://kiravillarreal.github.io/intro-workbook/20FinalProject/img/web-design.png" alt="copywriting" />
+          <h4>Copywriting</h4>
+          <p>I can create content for your website and marketing materials. Me write good stuff. </p>
+          </article>
+          <article>
+          <img class="animated infinite pulse" src="https://kiravillarreal.github.io/intro-workbook/20FinalProject/img/print.png" alt="voiceovers" />
+          <h4>Commercial Voiceovers</h4>
+          <p>Planning a television or radio ad campaign? With over 20 years of experience behind a microphone, I can provide top-quality commercial voiceovers for all your advertising needs.</p>
+          </article>
+        </div>
+      </section>
+      <section class="portfolio" id="portfolio">
+        <h2>Portfolio</h2>
+        <h4>Check out some of my previous work</h4>
+        <div class="owl-carousel">
+          <div class="item"><a href="https://kiravillarreal.github.io/intro-workbook/15LessonFifteen/Checkpoint3/"><img src="https://kiravillarreal.github.io/intro-workbook/15LessonFifteen/Checkpoint3/img/wine-bg.jpg" alt="foundry"/></a></div>
```

Comment:
Probably want to add `target="_blank"` to the `a` elements here so the user isn't navigated away from your portfolio page.